Tiltfile

```diff
@@ -110,7 +110,7 @@ local_resource(
 
 # Note: we need a distro with a bash shell to exec into the Velero container
 tilt_dockerfile_header = """
-FROM ubuntu:focal as tilt
+FROM ubuntu:latest as tilt
```

Comment:
Is it valid to use `22.04` (or  `jammy`) instead of `latest` tag (it is the same thing today, but in the future will change)

Ref https://hub.docker.com/_/ubuntu

Reply:
@anshulahuja98 I think this is a valid suggestion, will you make the change?

Reply:
Done